plugins/emotes.py

```diff
@@ -117,14 +121,42 @@ class Emotes:
+            try:
+                r = requests.get(proposed_emote['url'])
+                imageBytes = r.content
+                if (imageBytes[:4] != b'\x89PNG' and imageBytes[:3] != b'GIF' and imageBytes[6:10] != b'JFIF'):
+                    await message.channel.send('That emote is not a valid file type! We can only use JPG, PNG, and GIF')
+                    return
+                await message.guild.create_custom_emoji(name=name, image=imageBytes, roles=[], reason='Voted in!')
+                await message.channel.send('Emote %s was added to the discord!' % name)
+                proposed_emotes.pop(name)
+            except:
+                await message.channel.send('Unable to get the emote image! Check that the link works')
+            return
+        except KeyError:
+            await message.channel.send('Emote `' + name + '` does not exist!  (...but you may add it) :thinking:')
+
+
     @staticmethod
     async def help(message, args, author, client):
         embed = discord.Embed(title="Emotes", colour=discord.Colour(0x1869f), description="Command options:\n\n")
-        embed.add_field(name="-a", inline="false", value="Add an emote to the list of proposed emotes\n```" + Emotes.EMOTE_ADD_SYNTAX + "```")
-        embed.add_field(name="-v", inline="false", value="Vote for a proposed emote\n```" + Emotes.EMOTE_REMOVE_SYNTAX + "```")
-        embed.add_field(name="-r", inline="false", value="Remove an emote from the list of proposed emotes\n```" + Emotes.EMOTE_VOTE_SYNTAX + "```")
-        embed.add_field(name="-rv", inline="false", value="Revoke your vote from one of the proposed emotes (note: 'v' and 'r' may be specified in any order, including but not limited to: `-vr`, `-rv`, `-v -r`, or `-r -v`)\n```" + Emotes.EMOTE_REVOKE_VOTE_SYNTAX + "```")
-        embed.add_field(name="-c", inline="false", value="View the server's current emotes (this one's just for fun)```" + Emotes.EMOTE_CURRENT_EMOTES_SYNTAX + "```")
+        embed.add_field(name="-a", inline="false", value="Add an emote to the list of proposed emotes\n```" + Emotes.EMOTE_ADD_SYNTAX() + "```")
+        embed.add_field(name="-v", inline="false", value="Vote for a proposed emote\n```" + Emotes.EMOTE_VOTE_SYNTAX() + "```")
+        embed.add_field(name="-r", inline="false", value="Remove an emote from the list of proposed emotes\n```" + Emotes.EMOTE_REMOVE_SYNTAX() + "```")
+        embed.add_field(name="-rv", inline="false", value="Revoke your vote from one of the proposed emotes (note: 'v' and 'r' may be specified in any order, including but not limited to: `-vr`, `-rv`, `-v -r`, or `-r -v`)\n```" + Emotes.EMOTE_REVOKE_VOTE_SYNTAX() + "```")
+        embed.add_field(name="-c", inline="false", value="View the server's current emotes (this one's just for fun)```" + Emotes.EMOTE_CURRENT_EMOTES_SYNTAX() + "```")
+        embed.add_field(name="-u", inline="false", value="Create an emote thats been proposed (bot admins only)```" + Emotes.EMOTE_CREATE_SYNTAX() + "```")

```

Comment:
```suggestion
        embed.add_field(name="-u", inline="false", value="Create an emote that's been proposed (bot admins only)```" + Emotes.EMOTE_CREATE_SYNTAX() + "```")
```